docs/concepts/hybrid-approach.md

```diff
@@ -76,6 +81,28 @@ Each entry contains:
 The properties `icm` and `pwaBuild` can use [named capture groups](<https://2ality.com/2017/05/regexp-named-capture-groups.html#replace()-and-named-capture-groups>) and are only used in the _node.js_ process running on the server.
 However, `pwa` and `icmBuild` are used in the client application where [named capture groups are not yet supported by all browsers](https://github.com/tc39/proposal-regexp-named-groups#implementations).
 
+## PWA Adaptions
+
+With version 0.23.0 the PWA was changed to no longer reuse the Responsive Starter Store application types but rather be based upon the newly introduced headless application type for REST Clients - `intershop.REST`.
+This application type would be completely independent of the Responsive Starter Store and is not suited for storefront setups with the Hybrid approach.
+For this reason the PWA needs to be adapted to work together with the Responsive Starter Store again.
+
+**Migration Steps to prepare the PWA for the Hybrid Approach with the Responsive Starter Store**
+
+- use a current PWA version
+- revert the following Git commits from Jun 23, 2020
+
+| commit                                                                                                | comment                                                         |
+| ----------------------------------------------------------------------------------------------------- | --------------------------------------------------------------- |
+| [d7504a7](https://github.com/intershop/intershop-pwa/commit/d7504a76c9cf6b944b86f2b4d0160083003b396b) | rename references to 'app_sf_responsive_cm' to 'app_sf_base_cm' |
+| [ddf89a7](https://github.com/intershop/intershop-pwa/commit/ddf89a7f7c0326c5b37e1b2a003db178d96e19c2) | demo content changes - dialogs                                  |
```

Comment:
in this commit, the modal for ready-for-shipment is set to lg, is this really part of the migration? 😉

Reply:
The commit is right, but it might not be a good idea to include the small modal dialog fix in this commit in the first place.
I will need to change that.

Reply:
Removed unneeded and unrelated changes and reduced the set of relevant commits to 2.